README.md

```diff
@@ -37,12 +38,35 @@ regenerate Go code after you change the `sourcegraph.proto` file.
    git checkout proto3
    go install ./...
    ```
-1. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
+
+3. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
 
    ```
    go get -u sourcegraph.com/sourcegraph/gen-mocks
    ```
 
+4. **Install `gopathexec`**:
+
+   ```
+   go get -u github.com/shurcooL/gopathexec
+   ```
+
+#### OS X-specific
+
+```
+brew install gnu-sed --with-default-names
```

Comment:
Really hard to know.

I agree this is a potentially invasive change that not everyone might want to do (if they depend on existing OS X-specific `sed` for some other things). The only alternatives I can think of:

1. Use a namespaced version of sed somehow. So if you remove the `--with-default-names` option, it uses `gsed` binary name. So we can remove the `--with-default-names` flag, and add Linux section that does `alias gsed='sed'` or so.

2. Don't use `sed` since it's not portable. Perhaps write a simple (portable) Go tool that does what we want (does not need to be general like sed, just enough to perform the needed task, which is simple).